### PR TITLE
fix Rollup config

### PR DESCRIPTION
**Pull request by Rich-Harris**
This PR implements the fixes explained in https://github.com/GoogleChrome/sample-framework-boot/pull/10#issuecomment-272632548

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -34,6 +34,7 @@
     "@angular/platform-browser": "2.3.0",
     "babel-cli": "^6.18.0",
     "babel-plugin-transform-react-jsx": "^6.8.0",
+    "babel-preset-es2015-rollup": "^3.0.0",
```
Comment (hzoo) [inline package.json]: @Rich-Harris Unrelated to the pr itself maybe we should make another preset that includes external helpers?

currently babel-preset-es2015 allows you to choose the modules plugin: http://babeljs.io/docs/plugins/preset-es2015/#options with `modules: false` to not compile es modules.

es2015 rollup is just `modules: false` + `external-helpers`. We could just add it to [preset-env](https://github.com/babel/babel-preset-env/)?

Comment (Rich-Harris) [reply package.json]: Sounds good. It would be better for people who use Rollup and Babel if there wasn't a separate preset they had to worry about (es2015-rollup dates from a time before `modules: false`), so the more we can coalesce around a standard way of doing things that doesn't depend on specific forks of specific presets the better

Comment (hzoo) [reply package.json]: > so the more we can coalesce around a standard way of doing things that doesn't depend on specific forks of specific presets the better

Yeah true that makes sense. This is the same issue I'm finding with preset-20xx in general. It doesn't seem necessary to have it anymore since it's rare for someone to only want a specific year anyway. es2015, 16,17 presets aren't that useful anymore when you can use preset-latest or preset-env. 
Otherwise then there will be a 2018, etc and there's already squatted pkgs for some of these haha (unless we switch to `@babel/preset-es20xx`

So basically we could also make it an option under preset-es2015 and under preset-env if that is :+1:.

Comment (Rich-Harris) [reply package.json]: Works for me, yep. When you say 'option', are you referring just to `modules` or also to `external-helpers`? (To clarify the reasoning behind `external-helpers`, since it's far from obvious — it allows us to create a synthetic module such that `babelHelpers.classCallCheck` etc become mangleable functions rather than properties of an object)

Comment (hzoo) [reply package.json]: modules is actually already an option under both presets so yeah `external-helpers` - could name the option differently as well.

**Comment by paullewis** [comment]
Hey, thanks for the PR! Would like to merge, but as of #10 (I think) this is conflicting. Any chance you could take a look?

**Comment by Rich-Harris** [comment]
Ha, sorry — should have been clearer, this PR was an *alternative* to #10. I've updated it, should be good to go now
